package.json

```diff
@@ -11,17 +11,18 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@types/dotenv": "^6.1.1",
     "@types/express": "^4.16.1",
```

Comment:
all the @types/* should be on devDependencies, not only @types/dotenv

Reply:
https://github.com/targeek/slm-data-exporter/blob/4a147a1fa5df8c52873e7fb00e0801ae0530b497/package.json#L14-L15